Developer.md

```diff
@@ -0,0 +1,22 @@
+# Building flyxc
+
+## Preparation
+
+Copy `keys.ts.dist` to `keys.ts` and fill it out (there are two such files)
```

Comment:
Would you mind expanding that to 2 bullets points with the full path to each of the file?